docs/operator-manual/ingress.md

```diff
@@ -198,6 +198,54 @@ ArgoCD endpoints may be protected by one or more reverse proxies layers, in that
 $ argocd login <host>:<port> --header 'x-token1:foo' --header 'x-token2:bar' # can be repeated multiple times
 $ argocd login <host>:<port> --header 'x-token1:foo,x-token2:bar' # headers can also be comma separated
 ```
+## ArgoCD Server and UI Root Path
+
+ArgoCD server and UI can be configured to be available under a non-root path (e.g. `/argo-cd`).
+To do this, add the `--rootpath` flag into the `argocd-server` deployment command:
+
+```yaml
+spec:
+  template:
+    spec:
+      name: argocd-server
+      containers:
+      - command:
+        - /argocd-server
+        - --staticassets
+        - /shared/app
+        - --repo-server
+        - argocd-repo-server:8081
+        - --rootpath
+        - /argo-cd
+```
+NOTE: The flag `--rootpath` changes both API Server and UI base URL. 
+Example nginx.conf:
+
+```
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+
+    sendfile on;
+
+    server {
+        listen 443;
+
+        location /argo-cd {
```

Comment:
```suggestion
        location /argo-cd/ {
```

I believe the right trailing slash is required in both `location` and `proxy_pass`.